core/src/test/scala/org/apache/spark/sql/delta/cdc/CDCReaderSuite.scala

```diff
@@ -133,6 +133,24 @@ class CDCReaderSuite
     }
   }
 
+  test("execute sizeInBytes for TahoeCDCBaseFileIndex") {
+    withTempDir { tempDir =>
+      val log = DeltaLog.forTable(spark, tempDir.toString)
+      import io.delta.implicits._
+      spark.range(10).write.delta(tempDir.toString)
+      writeCdcData(
+        log,
+        spark.range(1, 2).toDF.withColumn(CDC_TYPE_COLUMN_NAME, lit("update_pre")))
+      sql(s"DELETE FROM delta.`${tempDir.toString}`")
+      // The main purpose of the test is to execute `sizeInBytes`,
+      // which is called by the following expression.
+      val changesStats = CDCReader.changesToBatchDF(log, 0, 2, spark)
+        .queryExecution.analyzed.stats
+      val expectedStats = Statistics(3169)
```

Comment:
where did 3169 come from?

Reply:
Its the fixed size of analyzed stats sizeInBytes found every time for this specific query (changesDF). I've added the comment above to explain that I'm not trying to test the statistics, but rather kick that sizeInByte computation (to verify that there are no MatchExceptions as there was before, with the missing `AddCDCFile`.